trusted-types/support/attributes.js

```diff
@@ -284,9 +284,9 @@ const attributeSetterData = [
   {
     api:"Attr.value",
     acceptNS: true,
-      acceptTrustedTypeArgumentInIDL: false,
-      runSetter: function(element, attrNS, attrName, attrValue, type) {
-      element.setAttributeNS(attrNS, attrName, createTrustedOutput(type, ""));
+    acceptTrustedTypeArgumentInIDL: false,
+    runSetter: function(element, attrNS, attrName, attrValue, type, initialAttrValue = "") {
+      element.setAttributeNS(attrNS, attrName, createTrustedOutput(type, initialAttrValue));
```

Comment:
I see you are now adding an optional initialAttrValue. I think my original idea was to keep runSetter minimal so we can actually test situations when the element has no attributes and each test can decide to attach an attribute or not before running the setter. Arguably, finding the best generic APIs that works in all cases is a bit tricky...

If we do your proposed change, then we should update the documentation and make sure all versions of runSetter are consistent. I would also make the initialAttrValue optional so no attribute is attached by default (for these three cases the attribute would be mandatory though otherwise we don't have any attribute node). Common initialization can be factor out in a helper function.